src/aggregators/util.ts

```diff
@@ -49,6 +54,13 @@ export interface IHistogramEvent {
   observations: number[];
 }
 
+export interface IDistributionEvent {
+  metricType: "distribution";
+  metricName: string;
+  labels: { [key: string]: string };
+  observations: number[];
```

Comment:
an event can have multiple observations if 1 frontend client would have multiple observations?

Reply:
an event can have multiple observations if 1 frontend client would have multiple observations in the interval between sending events